packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -460,80 +538,105 @@ Accounts.prototype.getLegacyAccount = function getLegacyAccount(key) {
   return { legacyAccount: account, klaytnWalletKeyAddress }
 }
 
+/**
+ * signTransaction signs to transaction with private key.
```

Comment:
It seems that if we already have signatures in the tx, the new signature will be appended. If it is right, please describe more about that.

Reply:
The description for appending was added :)